launchable/utils/http_client.py

```diff
@@ -34,7 +35,12 @@ def request(self, method, path, **kwargs):
             raise Exception("unable to post to %s" % url) from e
 
     def _headers(self):
-        return {
+        h = {
             "User-Agent": "Launchable/{} (Python {}, {})".format(__version__, platform.python_version(), platform.platform()),
-            'Authorization': 'Bearer {}'.format(self.token)
+            'Authorization': 'Bearer {}'.format(self.token),
         }
+
+        if self.test_runner != "":
+            h["Launchable-Test-Runner"] = self.test_runner
```

Comment:
What do you think this field name?

Reply:
I recommend you to use `launchable-test-runner`.
I want to use a name that can also be used for http/2 for future reference.

https://tools.ietf.org/html/rfc7540#section-8.1.2
>  A request or response containing uppercase  header field names MUST be treated as malformed

Reply:
OK, I thought that I should arrange the name with other headers. but I'll use http2 rule